packages/widgets/src/blocks/widget-group/editor.scss

```diff
@@ -0,0 +1,39 @@
+.wp-block-widget-group {
+	&.has-child-selected::after {
+		border-radius: $radius-block-ui;
+		border: 1px solid var(--wp-admin-theme-color);
+		bottom: 0;
+		content: "";
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+	}
+
+	.widget-title {
+		font-family: $default-font;
+		font-size: 18px;
+		font-weight: 600;
+	}
+}
+
+.wp-block-widget-group__placeholder {
+	.block-editor-inserter {
+		width: 100%;
+	}
+}
```

Comment:
we should add some styles in here somewhere for the button inserter colour in dark themes, as currently it's invisible:
<img width="697" alt="Screen Shot 2021-09-07 at 9 55 09 am" src="https://user-images.githubusercontent.com/8096000/132268046-84bf1bce-100b-469f-928d-c1d172df59a8.png">

The styles we need to override are coming from [here](https://github.com/WordPress/gutenberg/blob/add/widget-group/packages/block-editor/src/components/button-block-appender/style.scss#L11-L12).

Reply:
Good catch!